public/pages/AnomalyCharts/components/FeatureChart/FeatureChart.tsx

```diff
@@ -143,17 +142,6 @@ export const FeatureChart = (props: FeatureChartProps) => {
     return undefined;
   };
 
-  const getFeatureAnnotations = () => {
-    if (isEmpty(props.annotations)) {
-      return [];
-    }
-    return filterWithDateRange(
-      props.annotations,
-      props.dateRange,
-      'coordinates.x0'
-    );
-  };
-
```

Comment:
not needed anymore, since all filtering is done prior to being passed to this component, which should be stateless.